index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>About Me</title>
+  </head>
+  <body>
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home"><img src="./images/logo.png" alt="Home"></a>
+        <div id="myLinks">
+          <a href="#contact">Contact</a>
+          <a href="#about">About</a>
+        </div>
+        <a href="javascript:void(0);" class="icon" onclick="toggleNav()">
+			<img src="./images/hamburger.png" alt="Menu icon">
+        </a>
+      </nav>
+
+      <div class="content">
+        <h1>I am Siddhartha Glad to see you!</h1>
+        <p>I am a software developer!. I can help you build a product, feature or website.
+			 Look through some of my work and experience! 
+			 If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <p></p>
```

Comment:
- [ ] Kindly consider giving this paragraph the right color as the one given in the Figma design to have an exact duplicate of the provided design and also remember to remove the second opening <p> under the paragraph to make  to follow the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
@divinecharlotte since for the last 5 years i have been working as a backend engineer my knowledge in user end is very little so i cant figure out the colour of the text that i should apply here. Please help me out

Reply:
@divinecharlotte Eureka! i have changed the font and colour of the text of this section according to the way that you wanted